environment.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
   - pandas
   - pre-commit
   - psutil
+  - pyirf
```

Comment:
Can you also add this to `ctapipe/tools/info.py` in the list of dependency version to print? (`_dependencies`)

Reply:
done